### PR TITLE
PMM-7677 ability to run version service next to PMM Server

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

**Review by puneet0191 — APPROVED** [review]

**Review by nikita-b — APPROVED** [review]

File: pmm/aws-staging-start.groovy [hunk 1]
```diff
@@ -81,6 +81,12 @@ pipeline {
             choices: ['perfschema', 'slowlog'],
             description: "Query Source for Monitoring",
             name: 'QUERY_SOURCE')
+        string(
+            defaultValue: '',
+            description: '''
+            Docker image for version service, needs if you want to run your own version service.
```
Comment (atymchuk) [inline pmm/aws-staging-start.groovy:87]:
```suggestion
            Docker image for version service, use it if you want to run your own version service.
```

Comment (atymchuk) [reply pmm/aws-staging-start.groovy:87]: nit

File: pmm/aws-staging-start.groovy [hunk 2]
```diff
@@ -81,6 +81,12 @@ pipeline {
             choices: ['perfschema', 'slowlog'],
             description: "Query Source for Monitoring",
             name: 'QUERY_SOURCE')
+        string(
+            defaultValue: '',
```
Comment (atymchuk) [inline pmm/aws-staging-start.groovy:85]: Can we have a default value here, something like "latest" or "dev-latest"?

**Review by atymchuk — APPROVED** [review]

File: pmm/aws-staging-start.groovy (hunk 2 above)
Comment (BupycHuk) [reply pmm/aws-staging-start.groovy:85]: no, because usually we don't want to run version service next to PMM Server.

**Pull request merged**